scripts/cluster_providers/kind_wait_for_cluster

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+kubectl config set-context kind-$CLUSTER_VERSION
```

Comment:
should receive cluster name as arg, and based on that set the context if necessary. Also, it looks like that prev logic is a bit more complex:

https://github.com/helm/acceptance-testing/blob/master/lib/Kind.py#L61-L86

and it looks that we should implement it.